test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -65,3 +73,23 @@ TestCase::TestResult SMTCheckerTest::run(ostream& _stream, string const& _linePr
 
 	return conclude(_stream, _linePrefix, _formatted);
 }
+
+void SMTCheckerTest::filterObtainedErrors()
+{
+	SyntaxTest::filterObtainedErrors();
+
+	static auto removeCex = [](vector<SyntaxTestError>& errors) {
+		for (auto& e: errors)
+			if (
+				auto cexPos = e.message.find("\\nCounterexample");
+				cexPos != string::npos
+			)
+				e.message = string(e.message.begin(), e.message.begin() + static_cast<long int>(cexPos));
```

Comment:
```suggestion
				e.message = e.message.substr(0, cexPos);
```
maybe? Not too important.

Reply:
yea ofc